app/models/course_lesson.rb

```diff
@@ -0,0 +1,13 @@
+class CourseLesson < ApplicationRecord
```

Comment:
Couldn't we just put `belongs_to :course, optional: true, class_name: 'Course'` on `Lesson` and skip this entire join table? seeing as a lesson can only be in one Course.